go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/c2fo/vfs/v5
+module github.com/c2fo/vfs/v6
 
 go 1.13
```

Comment:
V6 might be a great time to update this to at least 1.16 since old 1.15 and older are no longer supported (by google) versions.

Reply:
Updated in this commit https://github.com/C2FO/vfs/pull/102/commits/ce88dce550f3ef7640bd83c8b18fbc6f19fb8dc7